### ISSUE TITLE
"dataquality/dataplatform" not working as expected

### ISSUE THREAD

**Issue by bsieber-mozilla** [mozilla/jira-bugzilla-integration#479]
Slack message: https://mozilla.slack.com/archives/C016BTJKM9B/p1682060585086999

**Comment by bsieber-mozilla** [comment]
~I believe there is some issue with the way we are passing jira_components in production?~

 <img width="409" alt="Screenshot 2023-04-21 at 2 54 54 PM" src="https://user-images.githubusercontent.com/53873986/233741147-7d35f354-8c54-40ed-8d7b-950d3cf45056.png">
----

I tried to reproduce this issue in staging (adding the components in question to the JB project) but it was able to process in staging:
https://bugzilla-dev.allizom.org/show_bug.cgi?id=1633577

**Comment by scholtzan** [comment]
Hey, just wondering if there has been any update on this issue?

**Comment by bsieber-mozilla** [comment]
Hey Anna, just reached out on Slack--moving the comment here:

> Hey Anna, it seems debugging this has taken awhile thus far--I believe this has to do with either the components having differing access levels than the main project (and the service account doesnt have permissions) or the status’s aren’t mapping.  (Is that something you can confirm/ provide a
> What I suggest for now, to get this data flowing--we remove the component mapping; the “Components” and routing/adding this data can still be added. 
> [@Julien](https://mozilla.slack.com/team/USA0HULBX)
>  could inform you a bit about how Jira Automation fills in some of the edges for this sync system.

**Comment by bsieber-mozilla** [comment]
In the above comments, I point out how this problem is not reproducible in stage--I've opened a JSM request to ensure that the Service Account has the right permissions for production.

**Comment by grahamalama** [comment]
@scholtzan, following up on this:

We enabled some new logging in #503, and we're seeing a possible source of this issue:

`"labels": "Field 'labels' cannot be set. It is not on the appropriate screen, or unknown."`
Source: https://mozilla.sentry.io/issues/3571741650/?project=6364263

For now, we're going to disable the `sync_whiteboard_labels` feature ([docs](https://github.com/mozilla/jira-bugzilla-integration/blob/main/docs/actions.md#default)) to hopefully unblock the queue. This means that the whiteboard tags on the bug won't be added as labels on the issue. 

We _think_ the error message that Jira is responding with means that there isn't an option to add labels on the "new issue" screen of the DENG project. If you'd like to reenable this feature, we'll want to make sure that field is there. @jchaupitre, what are your thoughts about our hunch of the `issues` field missing?

**Comment by scholtzan** [comment]
Would it make sense to revert https://github.com/mozilla/jira-bugzilla-integration/pull/491 to make sure bugs land in the right component? After removing the component config we set up a Jira automation based on the labels, which will now not be set.

**Comment by grahamalama** [comment]
Ahh I see. We can revert #491, or we can verify that Jira labels can be set on new issues, in which case we can flip `sync_whiteboard_labels` back to `True`. Do you have a preference either way?

**Comment by scholtzan** [comment]
Using components would be my preference

**Comment by grahamalama** [comment]
@scholtzan and I chatted about this on Slack but, it looks like this is working as expected.

`https://bugzilla.mozilla.org/show_bug.cgi?id=1833668` triggered the creation of `https://mozilla-hub.atlassian.net/jira/software/c/projects/DENG/issues/DENG-892`, which included the component being set.

Closing this issue as resolved. @scholtzan, feel free to reopen if you see further issues.
